src/components/networks/networkOverviewTab.jsx

```diff
@@ -132,12 +157,31 @@ export const NetworkOverviewTab = ({ network }) => {
                         <DescriptionListDescription id={`${idPrefix}-ipv6-prefix`}> {ip[1].prefix} </DescriptionListDescription>
                     </DescriptionListGroup> }
 
-                    { ip[1].dhcp.range.start && <DescriptionListGroup>
-                        <DescriptionListTerm> {_("DHCP range")} </DescriptionListTerm>
-                        <DescriptionListDescription id={`${idPrefix}-ipv6-dhcp-range`}> {ip[1].dhcp.range.start + " - " + ip[1].dhcp.range.end} </DescriptionListDescription>
-                    </DescriptionListGroup> }
+                    <Text component={TextVariants.h5}>
+                        {_("DHCP Settings")}
+                    </Text>
+
+                    { ip[1].dhcp.range.start &&
+                    <DescriptionListGroup>
+                        <DescriptionListTerm> {_("Range")} </DescriptionListTerm>
+                        <DescriptionListDescription id={`${idPrefix}-ipv6-dhcp-range`}>
+                            {ip[1].dhcp.range.start + " - " + ip[1].dhcp.range.end}
+                        </DescriptionListDescription>
```

Comment:
Duplicated code -- I think this is just a leftover from the IPv6 support, which you intended to remove? Or do you want to keep the *display* of existing IPv6 mappings, and just not edit them?

If you want to keep it, please move this into some `StaticDHCPSettings` component and parameterize it for IPv4/6.